Microsoft.Toolkit.Uwp.UI/Extensions/Tree/VisualTree.cs

```diff
@@ -78,6 +78,37 @@ public static T FindDescendant<T>(this DependencyObject element)
             return retValue;
         }
 
+        /// <summary>
+        /// Find first descendant control of a specified type.
+        /// </summary>
+        /// <param name="element">Parent element.</param>
+        /// <param name="type">Type of descendant.</param>
+        /// <returns>Descendant control or null if not found.</returns>
+        public static object FindDescendant(this DependencyObject element, Type type)
```

Comment:
Why is this needed? What'd wrong with 
`public static T FindDescendant<T>(this DependencyObject element)`

Reply:
If you don't know the type at compile time you can't call the generic method.

Reply:
Why would you need this if you don't know what type you would be looklng for

Reply:
The FindAncestor doesn't have the type at compile time as it's executed through XAML.  It uses the Ascendant method though, I added this for completeness.